README.md

```diff
@@ -226,27 +226,60 @@ The VC++ linker supports `/SOURCELINK` [switch](https://docs.microsoft.com/en-us
+- Not supported by [Azure DevOps Artifacts](https://azure.microsoft.com/en-us/services/devops/artifacts) service.
+
+If a .snupkg does not work for your scenario, consider including debug information in the main package via one of the alternatives.
+
+### Include in main package
+
+You can include PDB files in the main NuGet package by setting the following property in your project:
 
 ```xml
 <PropertyGroup>
   <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
 </PropertyGroup>
 ```
 
-Keep in mind that including PDBs in the .nupkg increases the size of the package and thus restore time for projects that consume your package, regardless of whether the user needs to debug through the source code of your library or not.
+> [!IMPORTANT]
+> Keep in mind that including debug information in the .nupkg increases the size of the package and thus restore time for projects that consume your package, regardless of whether the user needs to debug the source code of your library or not.
 
-.snupkg symbol packages have following limitations:
+> [!IMPORTANT]
+> When including PDB files in the main package, projects that consume the package must _also_ opt-in to copying the symbols into their own output directory. Starting in .NET 7 this can be controlled via the [`CopyDebugSymbolFilesFromPackages`](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#copydebugsymbolfilesfrompackages) property.
 
-- They do not support Windows PDBs (generated by VC++, or for managed projects that set build property `DebugType` to `full`)
-- They require the library to be built by newer C#/VB compiler (Visual Studio 2017 Update 9).
-- The consumer of the package also needs Visual Studio 2017 Update 9 or newer.
-- Not supported by [Azure DevOps Artifacts](https://azure.microsoft.com/en-us/services/devops/artifacts) service.
+> [!TIP]
+> Classic .NET projects should also consider changing the `DebugType` property to `portable` (or `embedded`) to match .NET SDK projects.
+
+### Embed in assembly
+
+You can embed Portable PDB debug information directly in the assembly by setting the following property in your project:
+
+```xml
+<PropertyGroup>
+  <DebugType>embedded</DebugType>
+</PropertyGroup>
+```
 
-Consider including PDBs in the main package if it is not possible to use .snupkg for the above reasons. 
-For managed projects, consider switching to Portable PDBs by setting `DebugType` property to `portable`. This is the default for .NET SDK projects, but not classic .NET projects.
+> [!IMPORTANT]
+> Keep in mind that embedding debug including debug information in assembly both increases the size of the package and thus restore time for projects that consume your package, regardless of whether the user needs to debug the source code of your library or not, and also may impact assembly load performance for very large assemblies.
 
```

Comment:
... and size of applications that deploy the binaries.

Reply:
Added!

Reworded this section a bit to make it clearer as well.

Reply:
I believe I've addressed your concerns, so marking this thread as Resolved. If you disagree or have any questions, please reactivate and let me know. Thanks!